### ISSUE TITLE
Boilerplate includes un-necessary shebang line

### ISSUE THREAD

**Issue by oz123** [kubernetes-client/python-base#154]
Currently the following modules contain this line:

```
watch/watch.py:#!/usr/bin/env python
watch/__init__.py:#!/usr/bin/env python
watch/watch_test.py:#!/usr/bin/env python
config/incluster_config_test.py:#!/usr/bin/env python
config/incluster_config.py:#!/usr/bin/env python
config/dateutil.py:#!/usr/bin/env python
config/kube_config_test.py:#!/usr/bin/env python
config/exec_provider.py:#!/usr/bin/env python
config/kube_config.py:#!/usr/bin/env python
config/__init__.py:#!/usr/bin/env python
config/config_exception.py:#!/usr/bin/env python
config/exec_provider_test.py:#!/usr/bin/env python
config/dateutil_test.py:#!/usr/bin/env python
stream/stream.py:#!/usr/bin/env python
stream/ws_client.py:#!/usr/bin/env python
stream/ws_client_test.py:#!/usr/bin/env python
stream/__init__.py:#!/usr/bin/env python
```

We should remove the shebang line from `hack/boilerplate/boilerplate.py.txt` and then remove it from the modules which aren't expected to run as scripts.

**Comment by micw523** [comment]
See discussion in #107 - this has been discussed

**Comment by oz123** [comment]
Yes, I see that this was discussed. And I also see that you made this comment in the past:

> I don’t think the python shebang should appear in the non-executable files. This applies to most files you added the shebang.

What is correct for the main repository isn't blindly correct to a python software. The main repository contains Python scripts which are predestined to be run by CI jobs and developers. For this case it is easier to type `./hack/script-name.py` instead of explicitly calling python before the script name.  The main (k/k) repository contains:

```
$ cd ~/Software/kubernetes
$ find . -type f -name "*.py"
./hack/update_owners.py
./hack/boilerplate/test/fail.py
./hack/boilerplate/test/pass.py
./hack/boilerplate/boilerplate_test.py
./hack/boilerplate/boilerplate.py
./hack/verify-flags-underscore.py
./hack/verify-publishing-bot.py
./translations/extract.py
```

For this repository there no point adding the shebang in Python modules and packages. This looks bad and conveys bad code smell (which might decrease usage in favor for other clients and eventually distract potential contributions).

IMHO these should be removed.

**Comment by scottilee** [comment]
+1

**Comment by oz123** [comment]
So, there seems to be agreement on this here, I will prepare a PR for this.

**Comment by yliaog** [comment]
thanks @oz123 please file a pr for this

**Comment by oz123** [comment]
@yliaog  I created #155 for this, please see if this is OK.

**Comment by roycaihw** [comment]
fixed in #155

/close
